bspump/elasticsearch/sink.py

```diff
@@ -35,8 +35,9 @@ def __init__(self, app, pipeline, connection, id=None, config=None, bulk_class=E
 		self.BulkClass = bulk_class
 
 		self.Index = self.Config.get('index')
-		if self.Index is None or len(self.Index) == 0:
-			L.warning("The 'index_prefix' has been renamed to 'index', adjust the configuration.")
+
+		if self.Index == "bspump_" and self.Config.get('index_prefix') != "bspump_" and len(self.Config.get('index_prefix')) > 0:
```

Comment:
Please add a comment that this is for "legacy" reasons.

Reply:
sure, thanks Ales.